### ISSUE TITLE
Set colors for new tab page in firefox theme

### ISSUE THREAD

**Issue by adrianinsaval** [Mrcuve0/Aritim-Dark#15]
Hi, would you mind adding the field to set the colors of the new tab page to your firefox theme? It looks out of place being white.

Here an example created with firefox color extension:
[theme.zip](https://github.com/Mrcuve0/Aritim-Dark/files/6023476/theme.zip)

```
      "ntp_background": "rgb(16, 21, 26)",
      "ntp_text": "rgb(211, 218, 227)",
```

these are the fields I'm talking about. Thanks!

**Comment by Mrcuve0** [comment]
Thank you! I published version 4.0 integrating your changes and refreshing the theme with the latest colors! Will be available as soon as it is approved by Mozilla.

[theme2.zip](https://github.com/Mrcuve0/Aritim-Dark/files/6096122/theme2.zip)

**Comment by adrianinsaval** [comment]
Thanks! 😁
